src/model/OrganizationSubscription.js

```diff
@@ -10,6 +11,18 @@ import Subscription from "./Subscription";
 const url = "/organizations/:organizationId/subscriptions/:id";
 
 export default class OrganizationSubscription extends Subscription {
+  constructor(subscription) {
+    const { api_url } = getConfig();
+
+    super(subscription);
```

Comment:
You can add the URL in the Subscription constructor, or set this._queryUrl aftrer super, I don't think we need an other _urlOverride var

Reply:
You cannot override a property starting with an underscore currently. Thomas and I got on a call to look over it and weren't sure the reason for that limitation and didn't want to touch that in case it broke things elsewhere. If you're good changing the Ressource handler to allow overriding underscore properties somehow, we can do that instead.

Reply:
You can override any property

Reply:
Thomas and I tested it on a video call, and trying to override `this._queryUrl` after calling `super(subscription)` didn't change `this._queryUrl` when `this` was inspected using `debugger`. Looking at the handler used in the constructor for `Ressource` shows that it places the new value in `this.data._queryUrl` instead of `this._queryUrl`, but prioritizes `this._queryUrl` in the getter, thus returning the original value instead of the overridden one.

The commit you added works fine for me, but if you want to clarify how overriding `this._queryUrl` is supposed to work, feel free to explain it.